frontend-js/style.css

```diff
@@ -93,3 +93,31 @@ a {
 .app-auth-btn {
   margin: 0 auto;
 }
+
+.categories-page {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+}
+.categories-title {
+  text-align: center;
+}
+.category-name {
+  text-align: center;
+  text-decoration-style: double;
```

Comment:
category-name does not have any text decoration, so this rule is applied to nothing?

Reply:
of course=)